homeassistant/components/tplink/light.py

```diff
@@ -214,7 +228,7 @@ def update(self):
                     # device returned no daily/monthly history
                     pass
 
-            self._available = True
+                self._available = True
```

Comment:
This will break it for bulbs which have no emeter (although I think all of them have it..)?